lualib/dkim_sign_tools.lua

```diff
@@ -37,6 +37,8 @@ local function prepare_dkim_signing(N, task, settings)
     rspamd_logger.debugm(N, task, 'mail is from address in sign_networks')
   elseif settings.sign_local and is_local then
     rspamd_logger.debugm(N, task, 'mail is from local address')
+  elseif settings.sign_received and not is_local and not auser then
```

Comment:
Probably, it is better to call it `sign_inbound`?

Reply:
That sounds better, I have changed it